### DISCUSSION TITLE
Error in "make check" with cpputest 3.8 in cygwin

### DISCUSSION THREAD

**Discussion by jiaBosch** [cpputest/cpputest#1757] [Q&A]
I downloaded cpputest 3.8 and tried to build in cygwin. When I followed instruction to use make check, I got following errors

FAIL: CppUTestTests.exe
PASS: CppUTestExtTests.exe
============================================================================
Testsuite summary for CppUTest 3.8
============================================================================
# TOTAL: 2
# PASS:  1
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
Please report to https://github.com/cpputest/cpputest
============================================================================
make[2]: *** [Makefile:2914: test-suite.log] Error 1
make[2]: Leaving directory '/cygdrive/c/work/leij/unittestingtool/cpputest-3.8'
make[1]: *** [Makefile:3022: check-TESTS] Error 2
make[1]: Leaving directory '/cygdrive/c/work/leij/unittestingtool/cpputest-3.8'
make: *** [Makefile:3236: check-am] Error 2

because the error, MakefileWorker.mk is not generated. I do not know how to continue, please help.

**Comment by orion2101** [comment]
Same problem. Have you come up with a solution ?

**Comment by basvodde** [comment]
Did you try it from the master?
